conclave-enclave/src/main/kotlin/com/r3/conclave/enclave/Enclave.kt

```diff
@@ -481,12 +469,15 @@ abstract class Enclave {
      * Handler which services requests from the host for Conclave EnclaveInstanceInfo attestation quotes.
      */
     private inner class GetEnclaveInstanceInfoQuoteCallHandler : CallHandler {
+        private var _quotingEnclaveInfo: ByteCursor<SgxTargetInfo>? = null
         private var _mostRecentQuote: ByteCursor<SgxSignedQuote>? = null
+        val quotingEnclaveInfo: ByteCursor<SgxTargetInfo> get() = checkNotNull(_quotingEnclaveInfo)
         val mostRecentQuote: ByteCursor<SgxSignedQuote> get() = checkNotNull(_mostRecentQuote)
 
         override fun handleCall(parameterBuffer: ByteBuffer): ByteBuffer? {
-            val quotingEnclaveInfo = ByteCursor.slice(SgxTargetInfo, parameterBuffer)
-            val quote = createAttestationQuote(quotingEnclaveInfo, createEnclaveInstanceInfoReportData())
+            _quotingEnclaveInfo = ByteCursor.slice(SgxTargetInfo, parameterBuffer)
+            print("GetEnclaveInstanceInfoQuoteCallHandler _quotingEnclaveInfo $_quotingEnclaveInfo")
```

Comment:
delete

Reply:
SOrry, this was to debug an integration test. Deleted now